content/blog/2022-06-14-DVC-VS-Code-extension.md

```diff
@@ -0,0 +1,159 @@
+
+<video controlslist="nodownload" preload="metadata" autoplay muted loop style="width:100%;">
+  <source src="/uploads/images/2022-06-14/run-experiment.mp4" type="video/mp4">
+Your browser does not support the video tag.
+</video>
+
+_Quickly run new ML experiments and view them in the experiments table._
+
+<video controlslist="nodownload" preload="metadata" autoplay muted loop style="width:100%;">
+  <source src="/uploads/images/2022-06-14/compare-experiments.mp4" type="video/mp4">
+Your browser does not support the video tag.
+</video>
+
+_Compare experiments with each other._
+
+<video controlslist="nodownload" preload="metadata" autoplay muted loop style="width:100%;">
+  <source src="/uploads/images/2022-06-14/live-metrics.mp4" type="video/mp4">
+Your browser does not support the video tag.
+</video>
+
+_Live updates to plots when using DVCLive._
+
+## Source control management
+
+On top of Git version control for your code, the source control view now lets
+you manage datasets and models tracked by DVC without using the terminal. The
+DVC panel shows you the state of the workspace, similarly to the familiar Git
+panel. From here you can track artifacts and synchronize versions with your
+remote repository.
+
+![Get a quick overview of changes to your data in the source control
+panel](/uploads/images/2022-06-14/source-control.png)
+
+## DVC tracked explorer
+
+Resources tracked by DVC can be viewed and managed through the tracked explorer
+view. See at a glance which datasets and models have been changed, and navigate
+through all files contained in the DVC project(s) of your workspace.
+
+## Command palette
+
+The DVC extension makes extensive use of the Command Palette, reducing the need
+to learn syntax by heart. Want to run a new experiment or pull data from your
+remote repository? Give the word in the Command Palette and the DVC extension
+will guide you along all required details.
+
+---
+
+https://media.giphy.com/media/cEYFeE4wJ6jdDVBiiIM/giphy.gif
+
+# What's next?
+
+We're excited that our VS Code extension is live and available to the public!
+From here on out, we plan on making the extension even better with new features
+such as pipeline (DAG) support,
+[TPI](https://github.com/iterative/terraform-provider-iterative) integration for
+remote execution, autocomplete for `dvc.yaml`, and parallel coordinate plots.
+
+Of course, we'd love to hear what you are most looking forward to. Make sure to
+give us feedback on what you'd like to see next!
+
+# Thank you! ❤️
+
+We would like to sincerely thank everyone who has helped make this project
+possible:
+
+- [Henning Dieterichs](https://github.com/hediet), for helping us get started
+- [Paige Bailey](https://twitter.com/DynamicWebPaige), for her support and warm
+  tweets
+- [Sid Unnithan](https://www.linkedin.com/in/siddhanthunnithan/), for his review
+  and help in getting the word out there
+- The VS Code developer community
```

Comment:
can we put a link? https://vscode-dev-community.slack.com/join/shared_invite/zt-zq9w7ddw-VD1NVQ4p2XLT7vh_kO7bJA#/shared-invite/email

Reply:
Thanks, was looking for one!